QUBEKit/run.py

```diff
@@ -724,11 +726,9 @@ def mm_optimise(molecule):
                            f'{molecule.constraints_file if molecule.constraints_file is not None else ""}',
                            shell=True, stdout=log, stderr=log)
 
-                # This will continue even if we don't converge this is fine
-                # Read the xyz traj and store the frames
-                molecule.read_file(f'{molecule.name}_optim.xyz', input_type='traj')
-                # Store the last from the traj as the mm optimised structure
-                molecule.coords['mm'] = molecule.coords['traj'][-1]
+                _, _, _, coords = read_input(Path(f'{molecule.name}_optim.xyz'))
```

Comment:
this feels super awkward that we have to load a function from a different file to put the data into the class and should be hidden behind some privet method. A lot could go wrong here if you change the return in future of read_input, and objects could be put in the wrong location if we get the order wrong like
```python
self.atoms , self.name, self.topology. self.coords = read_input(filename)
```

or just abstract this process so there is an easy way to do it as this will be repeated many times

```python
mol.add_conformer(filename)
```